pyvista/plotting/plotting.py

```diff
@@ -159,6 +153,9 @@ class BasePlotter(PickingHelper, WidgetHelper):
         The default is a Light Kit (to be precise, 5 separate lights
         that act like a Light Kit).
 
+    theme : pyvista.Theme, optional
```

Comment:
Last `pyvista.Theme` to be handled. Along with the corresponding error message below. OK, and the same for `Plotter()` too...

Reply:
Fixed in e532b75c0bf2bd488721e22816464a44dc5c4073 and 1aeca3ae71567cf7404f8c66147e1bb8f0440190.